src/torchjd/backward/backward.py

```diff
@@ -2,9 +2,14 @@
 
 from torch import Tensor
 
-from ._transform import Diagonalize, EmptyTensorDict, Init, Jac, Store, make_aggregation
-from ._transform.strategy import UnifyingStrategy
-from .aggregation import Aggregator
+from torchjd._transform import Diagonalize, EmptyTensorDict, Init, Jac, Store, make_aggregation
+from torchjd._transform.strategy import UnifyingStrategy
+from torchjd.aggregation import Aggregator
+from torchjd.backward._utils import (
```

Comment:
yes, it works better, didn't know about it. BTW we get warning about using the package `_transform`, are we doing something wrong ?

Reply:
I think _transform should be located inside the backward package

Reply:
Since it's not part of our api, I think we can move it there and revert the change later on if it turns out that we need _transform for another package that is not inside of the backward package